news/2020-11-26-community-mentorship-program-winter-2021-signups-now-open.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Community Mentorship Program Winter 2021 Signups Now Open
+date: 2020-11-26 08:00:00 +0000
+---
+
+With holiday cheer and an upcoming new year, the 17th cycle of the Community Mentorship program is on its way! If you've ever found yourself wanting to expand your current knowledge and skills or help beginner mappers get started, this program will serve as a great opportunity for you!
+
+![](/wiki/shared/news/banners/community-mentorship-program.jpg)
+
+The Community Mentorship program serves as a place for upcoming mappers and modders to develop their skills under the guidance of more experienced mappers and modders in the community. 
+
+Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events (like mapping contests), hang out with fellow mentees, and more!
+
+If you're experienced in mapping and have a desire to take those who are new under your wing, then give joining us as Mentor a shot! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1188335), you'll be able to apply right away. If you're only just getting into mapping or are eager to hone your skills, then signing up as Mentee might just be your way to do it!
```

Comment:
```suggestion
If you're experienced in mapping and have a desire to take those, who are new, under your wing, then give joining us as a Mentor a shot! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1188335), you'll be able to apply right away. If you're only just getting into mapping or are eager to hone your skills, then signing up as a Mentee might just be your way to do it!
```

Reply:
Really odd comma placement imo. Reads like "those" is a group of people on its own and "who are new" is a specification for them, even though "those" would have nothing to refer to and loses its meaning. Should together stay as is.
As Mentor/Mentee reads more naturally to me ? Should be okay.